rsciio/tests/test_emd_prismatic.py

```diff
@@ -20,9 +20,11 @@
 
 import numpy as np
 import pytest
+
+hs = pytest.importorskip("hyperspy.api", reason="hyperspy not installed")
+
 import traits.api as t
```

Comment:
remaining traits dependency (`rsciio` does not depend on `traits`, but I guess it is covered if `hs` is imported)